python/cugraph-dgl/pyproject.toml

```diff
@@ -3,8 +3,9 @@
 [build-system]
 
 requires = [
-    "setuptools>=61.0.0",
-    "wheel",
+    "cmake>=3.26.4,!=3.30.0",
+    "cython>=3.0.0",
+    "ninja",
```

Comment:
Don't worry about these changes for build dependencies of the `cugraph-dgl` and `cugraph-pyg` wheels... we're not building wheels yet for `cugraph-pyg` / `cugraph-dgl` on this repo yet anyway.

In the next PR after this, I'll update those libraries to `rapids-build-backend` and make the appropriate splits in `dependencies.yaml`.

Reply:
Sounds good.